dev/ScrollBar/ScrollBar_themeresources.xaml

```diff
@@ -531,42 +488,53 @@
+                                        <DoubleAnimation Duration="{ThemeResource ScrollBarOpacityChangeDuration}" Storyboard.TargetName="VerticalTrackRect" Storyboard.TargetProperty="Opacity" To="0" />
+
+                                        <DoubleAnimation Duration="{ThemeResource ScrollBarOpacityChangeDuration}" Storyboard.TargetName="HorizontalSmallIncrease" Storyboard.TargetProperty="Opacity" To="0" />
+                                        <DoubleAnimation Duration="{ThemeResource ScrollBarOpacityChangeDuration}" Storyboard.TargetName="HorizontalLargeIncrease" Storyboard.TargetProperty="Opacity" To="0" />
+                                        <DoubleAnimation Duration="{ThemeResource ScrollBarOpacityChangeDuration}" Storyboard.TargetName="HorizontalLargeDecrease" Storyboard.TargetProperty="Opacity" To="0" />
+                                        <DoubleAnimation Duration="{ThemeResource ScrollBarOpacityChangeDuration}" Storyboard.TargetName="HorizontalThumb" Storyboard.TargetProperty="Opacity" To="0" />
+                                        <DoubleAnimation Duration="{ThemeResource ScrollBarOpacityChangeDuration}" Storyboard.TargetName="HorizontalSmallDecrease" Storyboard.TargetProperty="Opacity" To="0" />
+                                        <DoubleAnimation Duration="{ThemeResource ScrollBarOpacityChangeDuration}" Storyboard.TargetName="HorizontalTrackRect" Storyboard.TargetProperty="Opacity" To="0" />
+
+                                        <DoubleAnimationUsingKeyFrames Storyboard.TargetName="VerticalThumb" Storyboard.TargetProperty="Width" EnableDependentAnimation="True">
+                                            <SplineDoubleKeyFrame KeyTime="{ThemeResource ScrollBarContractDuration}" Value="{ThemeResource ScrollBarVerticalThumbMinWidth}" KeySpline="0,0,0,1" />
+                                        </DoubleAnimationUsingKeyFrames>
+
+                                        <DoubleAnimationUsingKeyFrames Storyboard.TargetName="VerticalThumbTransform" Storyboard.TargetProperty="X" EnableDependentAnimation="True">
+                                            <SplineDoubleKeyFrame KeyTime="{ThemeResource ScrollBarContractDuration}" Value="{ThemeResource ScrollBarThumbOffset}" KeySpline="0,0,0,1" />
+                                        </DoubleAnimationUsingKeyFrames>
+
+                                        <DoubleAnimationUsingKeyFrames Storyboard.TargetName="HorizontalThumb" Storyboard.TargetProperty="Height" EnableDependentAnimation="True">
+                                            <SplineDoubleKeyFrame KeyTime="{ThemeResource ScrollBarContractDuration}" Value="{ThemeResource ScrollBarHorizontalThumbMinHeight}" KeySpline="0,0,0,1" />
+                                        </DoubleAnimationUsingKeyFrames>
+
+                                        <DoubleAnimationUsingKeyFrames Storyboard.TargetName="HorizontalThumbTransform" Storyboard.TargetProperty="Y" EnableDependentAnimation="True">
```

Comment:
>EnableDependentAnimation [](start = 150, length = 24)

Can you add a comment explaining the EnableDependentAnimation so we know we can remove it later in winui3 when the bug is resolved?